services/tests/test_gql_services_query.py

```diff
@@ -48,3 +48,33 @@ def test_can_filter_according_to_exact_client_id_of_service(
 
     assert "errors" not in executed
     assert executed["data"] == {"services": {"edges": []}}
+
+
+def test_services_require_service_connections_except_profile_service(
+    profile_service, service, anon_user_gql_client
```

Comment:
At first I though that service was unused here. Removed it and tested and the test was successful. That is of course correct, but then it doesn't test that the requireServiceConnection is set properly.

Could the testing service be created deliberately in the function body or maybe an assert that there are more than one service? To make it fail if someone is as daft as me.

Reply:
Yes, perhaps using ServiceFactory explicitly makes it more clear.

Reply:
Changed to use ServiceFactory directly.

Also remembered to add the requirement for elevated privileges to the API documentation.